src/components/Sum.jsx

```diff
@@ -1 +1,18 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Sum({ a, b }) {
+  return (
+    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+  );
+}
```

Comment:
```suggestion
export const Sum = ({ a, b }) => <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>;
```

If your function returns result immediately you can use arrow function.